CHANGELOG.md

```diff
@@ -5,6 +5,51 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [3.8] - 2023-02-01
+### Added
+- `distance` method
+- `__contains__` method
+- `__getitem__` method
+- Goodman-Kruskal's Lambda A benchmark
+- Goodman-Kruskal's Lambda B benchmark
+- Krippendorff's Alpha benchmark
+- Pearson's C benchmark
+- 30 new distance/similarity
+	1. AMPLE
+	2. Anderberg's D
+	3. Andres & Marzo's Delta
+	4. Baroni-Urbani & Buser I
+	5. Baroni-Urbani & Buser II
+	6. Batagelj & Bren
+	7. Baulieu I
+	8. Baulieu II
+	9. Baulieu III
+	10. Baulieu IV
+	11. Baulieu V
+	12. Baulieu VI
+	13. Baulieu VII
+	14. Baulieu VIII
+	15. Baulieu IX
+	16. Baulieu X
+	17. Baulieu XI
+	18. Baulieu XII
+	19. Baulieu XIII
+	20. Baulieu XIV
+	21. Baulieu XV
+	22. Benini I
+	23. Benini II
+	24. Canberra
+	25. Clement
+	26. Consonni & Todeschini I
+	27. Consonni & Todeschini II
+	28. Consonni & Todeschini III
+	29. Consonni & Todeschini IV
+	30. Consonni & Todeschini V
+### Changed
+- `relabel` method sort bug fixed
+- `README.md` modified
+- Document modified
+- Test system modified
```

Comment:
Shouldn't we mention the changes in `compare` overall benchmarks and overall benchmarks weights?

Reply:
You are right!

- `Compare` overall benchmarks default weights updated

Good?

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/481/commits/f2c225a008d946de2e9be0a50f0d9a827b54f92f